templates/cases/inbox_messages.haml

```diff
@@ -200,10 +206,14 @@
       float: left;
       width: 50px;
     }
-    .message-contact {
+    .message-contact-default {
       float: left;
       width: 80px;
     }
+    .message-contact-urns {
+      float: left;
+      width: 130px;
```

Comment:
If the urns max length stays at 12 digits, I'll shift the message left a bit to 120px, like this:
![120px](https://cloud.githubusercontent.com/assets/9653693/20677131/c30f3704-b59a-11e6-8527-af575eae293d.png)
but otherwise, looking good!, +1 from my side